test/Numerics/DGMethods/integral_test_sphere.jl

```diff
@@ -193,7 +193,7 @@ function test_run(mpicomm, topl, ArrayType, N, FT, Rinner, Router)
         @test all(Router - Rinner .≈ aux[:, :, end, int_r_ind, end, :])
         # FIXME: Reverse integral is currently off by one cell width
         Δ = (Router - Rinner) / nvertelem
-        @test all(Router - Rinner - Δ .≈ aux[:, :, 1, rev_int_r_ind, 1, :])
+        @test all(Δ .≈ aux[:, :, 1, rev_int_r_ind, 1, :])
```

Comment:
@jkozdon I changed the `Router - Rinner - Δ` to `Δ` only, which turns out to be the same thing. 

This reverse integral is still off by one cell width. A hack would be 
```suggestion
        @test all(Δ .≈ aux[:, :, 1, rev_int_r_ind, 1, :]/2)
```
:joy: 
but I was trying to understand why the value of this integral was off by one cell width in the first place

Reply:
```suggestion
         @test all(Router - Rinner .≈ aux[:, :, 1, rev_int_r_ind, 1, :]/2)
```
 Didn't work?

Reply:
```julia
        #TODO: Update to test all faces (similar to `integral_test.jl`
        # Only the last value of the forward integral is currently correct
        @test all(Router - Rinner .≈ aux[:, :, end, int_r_ind, end, :])
        # Currently only the first value of the reverse integral is correct
        @test all(Router - Rinner .≈ aux[:, :, 1, rev_int_r_ind, 1, :])
        # All the `JcV` (line integral metrics) values should be `Δ / 2`
        Δ = (Router - Rinner) / nvertelem
        @test all(Δ .≈ 2grid.vgeo[:, Grids._JcV, :])
```